tests/ti_deps/deps/test_runnable_exec_date_dep.py

```diff
@@ -39,11 +41,56 @@ def test_exec_date_after_end_date(self):
         """
         If the dag's execution date is in the future this dep should fail
         """
-        ti = self._get_task_instance(
-            dag_end_date=datetime(2016, 1, 3),
```

Comment:
Let's include dag_end_date too in the test

Reply:
I am '            end_date=datetime(2016, 11, 5),' line 47